README.md

```diff
@@ -1,182 +1,176 @@
-<a href="https://github.com/eclipse-thingweb/node-wot/issues?q=label%3Aquestion+">question</a> |
-<a href="https://github.com/eclipse-thingweb/node-wot/issues?q=label%3A%22good+first+issue%22+">good first issue</a>
+For further information please refer to the official [W3C Web of Things](https://www.w3.org/WoT/) website.
 
 ### Table of Contents
 
 <!-- https://ecotrust-canada.github.io/markdown-toc/ -->
 
--   [License](#license)
+-   [Installation](#installation)
+    -   [As a library](#as-a-library)
+        -   [Node.js](#nodejs)
+        -   [Browser](#browser)
+    -   [As a CLI tool](#as-a-cli-tool)
+        -   [As a docker image](#as-a-docker-image)
+-   [Examples](#examples)
 -   [Implemented/supported features](#implementedsupported-features)
     -   [Protocol Support](#protocol-support)
     -   [MediaType Support](#mediatype-support)
--   [Prerequisites](#prerequisites)
-    -   [To use with Node.js](#to-use-with-nodejs)
-    -   [To use in a browser](#to-use-in-a-browser)
--   [How to get the library](#how-to-get-the-library)
-    -   [As a Node.js dependency](#as-a-nodejs-dependency)
-    -   [As a standalone application](#as-a-standalone-application)
-    -   [As a Docker image](#as-a-docker-image)
-    -   [As a browser library](#as-a-browser-library)
 -   [No time for explanations - show me a running example!](#no-time-for-explanations---show-me-a-running-example)
     -   [Using Node.js](#using-nodejs)
     -   [Using Docker](#using-docker)
     -   [Using a browser](#using-a-browser)
 -   [Online Things](#online-things)
--   [How to use the library](#how-to-use-the-library)
+-   [Documentation](#documentation)
     -   [The API](#the-api)
     -   [TD Tooling](#td-tooling)
     -   [Logging](#logging)
     -   [Install new/different versions of Node.js](#install-newdifferent-versions-of-nodejs)
+-   [Contributing](#contributing)
+-   [License](#license)
 
-## License
+## Installation
 
-Dual-licensed under both
+The framework can be used in two ways: as a library or as a CLI tool. In this section we will explain how to install the framework in both ways.
 
--   [Eclipse Public License v. 2.0](http://www.eclipse.org/legal/epl-2.0)
--   [W3C Software Notice and Document License (2015-05-13)](https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document)
+### As a library
 
-Pick one of these two licenses that fits your needs.
-Please also see the additional [notices](NOTICE.md) and [how to contribute](CONTRIBUTING.md).
+The framework is composed by different packages that users can use as they please. The core package is `@node-wot/core` and it is the only mandatory package to install. The other packages are bindings that allow the framework to communicate with different protocols and optionally we offer a set of tools in the `@node-wot/td-tools` package.
 
-## Implemented/supported features
+#### Node.js
 
-### Protocol Support
+> **Warning**: We no longer actively support Node.js version 16 and lower.
```

Comment:
Maybe we could format this using GitHub's relatively recent [alert syntax](https://github.com/orgs/community/discussions/16925)? :)

```suggestion
> [!WARNING]
> We no longer actively support Node.js version 16 and lower.
```